integrations/tensorflow/e2e/keras/iree_vision_test_suite.bzl

```diff
@@ -55,11 +56,13 @@ def iree_vision_test_suite(
         an iterable of dictionaries with the keys `models`, `datasets` and
         `backends`. Each key points to a string or iterable of strings
         specifying a set of models, datasets and backends that are failing.
-      external_weights:
-        a base url to fetch trained model weights from.
       tags:
         tags to apply to the test. Note that as in standard test suites, manual
         is treated specially and will also apply to the test suite itself.
+      url:
+        a base url to fetch non-keras trained model weights from.
+      use_external_weights:
+        whether or not to load model weights from keras or a supplied url.
```

Comment:
This is a bit confusingly worded because of the second "or". What about

```suggestion
        whether or not to load model weights from the web (either uses keras or
          the supplied url).
```